core/arch/arm/plat-marvell/armada7k8k/armada7k8k_core_pos_a64.S

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2017, Marvell International Ltd.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ * this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include <arm.h>
+#include <asm.S>
+
+	.globl get_core_pos
+
+FUNC get_core_pos , :
+	mrs x0, mpidr_el1
+	b   plat_marvell_calc_core_pos
+END_FUNC get_core_pos
+
+/*
```

Comment:
minor: indentation below

Reply:
Er...sorry, I don't quite follow

Reply:
There is a extra spaces to remove in the 4 lines below.
```patch
 /*
-   *  unsigned int plat_marvell_calc_core_pos(uint64_t mpidr)
-   *  Helper function to calculate the core position. 
-   *  With this function: CorePos = (ClusterId * 2) + CoreId
-   */
+  *  unsigned int get_core_pos_mpidr(uint32_t mpidr)
+  *  Helper function to calculate the core position. 
+  *  With this function: CorePos = (ClusterId * 2) + CoreId
+  */
```
(note: 32bit argument, use the generic label `get_core_pos_mpidr` as @jenswi-linaro recommends)